src/node_mgmt/commands.py

```diff
@@ -211,6 +218,65 @@ def readall(s):
     return result
 
 
+def holykell_sensor_address_7(node):
+    return _change_address_holykell(original_slave_id=1, target_slave_id=7)
+
+
+def holykell_sensor_address_8(node):
+    return _change_address_holykell(original_slave_id=1, target_slave_id=8)
+
+
+def _change_address_holykell(original_slave_id: int, target_slave_id: int) -> dict:
+    mod = minimalmodbus.Instrument(port=DEFAULT_SERIAL_DEV, slaveaddress=original_slave_id, debug=True)
+    mod.serial.baudrate = DEFAULT_SERIAL_BAUD_RATE
+    mod.serial.timeout = 3
+    result = {}
+
+    try:
+        # check that no device is already on target slave address
+        _read_holykell(mod, slave_id=target_slave_id)
+    except minimalmodbus.NoResponseError:
+        # no device detected on target slave, go ahead and set address
+        result = _set_address_holykell(mod, result, original_slave_id=original_slave_id,
+                                       target_slave_id=target_slave_id)
+    except Exception as e:
+        # if any other exception than NoResponseError is caught, the command must fail
+        logger.warning(f'Failed to check if slave {target_slave_id} available. Exception {e}')
+        result['Error'] = f'Failed to check if slave {target_slave_id} available. Exception {e}'
+    else:
+        logger.info(f'Slave ID {target_slave_id} already in use')
+        result['Error'] = f'Other device already detected on slave {target_slave_id}'
+
+    return result
+
+
+def _set_address_holykell(mod, result: dict, original_slave_id: int, target_slave_id: int) -> dict:
```

Comment:
Can we add a type annotation for `mod` (I think it should be `minimalmodbus.Instrument`)

Reply:
Definitely, just corrected that :)